front-end/src/screens/Post/Post.tsx

```diff
@@ -39,19 +37,17 @@ const Post = ( { className, data }: Props ) => {
 	);
 }
 
-// FIXME Should be typed
-const PostContent = ({ post } : any) => {
-	const { author, category, content, title, creation_date } = post;
+const PostContent = ({ post } : {post: PostFragment}) => {
+	const { author, category, content, creation_date, title } = post;
 
 	if (!author || !author.username || !content) return <div>Post not available</div>
 
 	return (
 		<>
 			<div className='post_info'>posted by <strong>{author.username}</strong> {moment.default(creation_date, 'YYYY-MM-DDTHH:mm:ss.SSS').fromNow()}</div>
 			<div className='text_muted divider'>&nbsp;&nbsp;|&nbsp;&nbsp;</div>
-			<div className='text_muted'>All discussions are off-chain.</div>
 			<h3>{title}</h3>
-			<div className='post_content'>{content}</div>
+			<ReactMarkdown className='post_content' source={content} />
```

Comment:
This is from master, I added the `post_content` class, but the styling is surely off now.